src/motor_hardware.cc

```diff
@@ -80,7 +111,7 @@ double   g_radiansRight = 0.0;
 // This utility opens and reads 1 or more bytes from a device on an I2C bus
 // This method was taken on it's own from a big I2C class we may choose to use later
 static int i2c_BufferRead(const char *i2cDevFile, uint8_t i2cAddr,
-                          uint8_t* pBuffer, int16_t chipRegAddr, uint16_t NumByteToRead);
+                          uint8_t* pBuffer, uint16_t NumBytesToRead);
```

Comment:
NumByteToRead was changed to NumBytesToRead, but that replacement isn't noted in any other place, is this param even used? This reminds me, we might want to take SonarQube and run it over most of our code to see if it catches anything unseen.